docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+            If the deployement is gone from the list of deployments, than this means it is finished.
+            As long as the deployment runs, the effect of that change operation is visible only partially.
+          body:
+            application/json:
+              example: !include examples/deployments_result.json
+        404:
+          description: No task found with this `app_id`.
+          body:
+            application/json:
+              example: |
+                { "message": "App '/not_existent' does not exist" }
+
+  /tasks:
+    ###
+    # Create application
+    #
+    get:
+      description: List all running tasks for application `app_id`.
+      is: [ secured ]
+      responses:
+        200:
+          description: The list of running tasks for application `app_id`.
+          body:
+            application/json:
+              example: !include examples/app_tasks.json
+            text/plain:
+              example: |
+                minecraft_survival-world	10013	srv7.hw.ca1.mesosphere.com:31756
+        404:
+          description: No task found with this `app_id`.
+          body:
+            application/json:
+              example: |
+                { "message": "App '/not_existent' does not exist" }
+
+    ###
+    # Create application
+    #
+    delete:
+      description: Kill tasks that belong to the application `app_id`
+      is: [ secured ]
+      queryParameters:
+        host:
+          description: all tasks of that application on the supplied slave are killed
+        scale:
+          type: boolean
+          description: If `scale=true` is specified, then the application is scaled down by the number of killed tasks.
+          default: false
+      responses:
+        200:
+          description: The list of tasks that has been killed.
+          body:
+            application/json:
+              example: !include examples/app_tasks.json
+        404:
+          description: No task found with this `app_id`.
+          body:
+            application/json:
+              example: |
+                { "message": "App '/not_existent' does not exist" }
+
+    /{task_id}:
+      ###
+      # Create application
+      #
+      delete:
+        description: Kill the task with ID `task_id` that belongs to the application `app_id`.
+        queryParameters:
+          scale:
+            type: boolean
+            description: If `scale=true` is specified, then the application is scaled down by the number of killed tasks.
+            default: false
+        responses:
+          200:
+            description: The tasks that has been killed.
+            body:
+              application/json:
+          404:
+            description: No task found with this task_id.
+            body:
+              application/json:
+                example: |
+                  {"message":"Task 'not-existing' does not exist"}
+
+  ###
+  # Create application
```

Comment:
While writing this, they help me find the single endpoints more easily. I corrected all - hope this helps also in the future.